### PR TITLE
comments challenge - chantal

### PR DESCRIPTION
**Pull request by chantal66**

### CODE REVIEW

File: source/addComment.js [hunk 1]
```diff
@@ -1,3 +1,32 @@
...
+            return true
+    }
+
+    function refresh() {
+        $('#form').toggle(!formcheck)
+    }
+
+    $('#form').submit( (event) =>  {
+
+        let comment = $('#form :input')[0].value;
```
Comment (kat3kasper) [inline source/addComment.js:21]: Could use `input[name=comment]` to be more specific

File: source/addComment.js [hunk 2]
```diff
@@ -1,3 +1,32 @@
 $(document).ready(function(){
-  // add your code here
-})
+    $('#form').hide();
+
+    $('#new_comment_button').on('click',() => {
+        $('#form').toggle();
+    });
+
+    function formcheck(comment, author) {
```
Comment (kat3kasper) [inline source/addComment.js:8]: Love that you extract out this error check into a function 👍

File: source/index.html [hunk 3]
```diff
@@ -23,6 +23,14 @@
     </ul>
 
     <button id="new_comment_button">New Comment</button>
+    <div class="form">
```
Comment (kat3kasper) [inline source/index.html:26]: Is there a way we could have added the form using jQuery rather than HTML?

**Review by kat3kasper — COMMENTED** [review]
Good job! 👍

File: source/index.html (hunk 3 above)
Comment (chantal66) [reply source/index.html:26]: thank you so much for the feedback 😄

**Pull request closed without merging**